cypress/e2e/Filters/NoTrials/noTrials.js

```diff
@@ -0,0 +1,144 @@
+/// <reference types="Cypress" />
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+// Steps for data entry in filters
+Given('user has typed {int} in the Age text input', (age) => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.get('.ctla-sidebar').find('input[type="number"]').clear().type(age);
+});
+
+Given('user has typed {string} in the Location by ZIP Code text input', (zipCode) => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.get('.ctla-sidebar').find('input[type="text"]').clear().type(zipCode);
+});
+
+Given('user has selected {int} miles for the Radius dropdown', (radius) => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.get('.ctla-sidebar').find('select.usa-select').select(radius.toString());
+});
+
+// URL validation
+Then('the page URL stays the same with the additional query parameter', () => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.url().should('include', 'a='); // Age filter parameter
+	cy.url().should('include', 'z='); // ZIP code parameter
+	cy.url().should('include', 'zr='); // Radius parameter - Fixed from 'r=' to 'zr='
+});
+
+// Content validation
+Then('intro text displays', () => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.get('.ctla-results__intro').should('be.visible');
+});
+
+Then('Trials 0 of 0 displays below the intro text', () => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.get('.ctla-results__count').should('contain', 'Trials 0 of 0');
+});
+
+Then('horizontal rule displays below results count', () => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.get('.ctla-results__divider').should('be.visible');
+});
+
+Then('No Trials Found message displays below horizontal rule', () => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.get('.ctla-results__no-results').should('be.visible');
+});
+
+Then('No Trials Found message displays below the page title', () => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.get('h1').next().find('p').should('be.visible');
+});
+
+// Filter value verification
+Then('Age parameter has {int} entered', (value) => {
+	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
+	cy.get('.ctla-sidebar').find('input[type="number"]').should('have.value', value.toString());
```

Comment:
instead of converting value to string, why dont you pass it as string initially in the feature file?

Age parameter has "2" entered
```
Then('Age parameter has {string} entered', (value) => {
	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
	cy.get('.ctla-sidebar').find('input[type="number"]').should('have.value', value);
	});
```